### PR TITLE
Fix drawLine & drawPoints won't use blendFunc bug

### PR DESCRIPTION
**Pull request by xiaofeng11**
CCDrawNode only call GL::blendFunc in onDraw, but miss calling GL::blendFunc in onDrawGLLine and onDrawGLPoint

### CODE REVIEW

**Comment by pandamicro** [comment]
[ci rebuild]

File: cocos/ui/UILayoutComponent.cpp [hunk 1]
```diff
@@ -667,7 +667,23 @@ namespace ui {
         _owner->setPosition(ownerPosition);
         _owner->setContentSize(ownerSize);
 
-        ui::Helper::doLayout(_owner);
+        if (typeid(*_owner) == typeid(PageView))
+        {
+            PageView* page = static_cast<PageView*>(_owner);
+            page->forceDoLayout();
+
+            Vector<Layout*> _layoutVector = page->getPages();
```
Comment (zilongshanren) [inline cocos/ui/UILayoutComponent.cpp:675]: I suggest to use `auto&` and `range for loop here`, it's modern c++ style and it's more clean and compact.

Comment (xiaofeng11) [inline cocos/ui/UILayoutComponent.cpp:675]: Fixed

**Comment by pandamicro** [comment]
@xiaofeng11 Please fix compilation issues

**Comment by zilongshanren** [comment]
@xiaofeng11  please fix win8.1 compile error.

**Comment by xiaofeng11** [comment]
@andyque  all compile error fixed

**Comment by zilongshanren** [comment]
@xiaofeng11  Thanks.

**Pull request merged**